### PR TITLE
Fixed git issues

### PR DESCRIPTION
**Pull request by msalo3**
Ok, round 2

### CODE REVIEW

File: lib/qb_integration/processors/payment_line_item.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+module QBIntegration
+  module Processor
+    class PaymentLineItem
+      include Helper
+      attr_reader :line
+
+       def initialize(line, linked_txn)
+        @line = line
+        @linked_txn = linked_txn
+      end
```
Comment (gorandev) [inline lib/qb_integration/processors/payment_line_item.rb]: Indentation is kinda confuse here and in some lines below

File: lib/qb_integration/services/payment.rb [hunk 2]
```diff
@@ -37,6 +37,23 @@ def find_by_id(id)
         vendor
       end
 
+      def find_by_updated_at(page_num)
+        raise MissingTimestampParam unless config["quickbooks_poll_stock_timestamp"].present?
+
```
Comment (gorandev) [inline lib/qb_integration/services/payment.rb]: Some weird indentation here as well.

File: quickbooks_endpoint.rb [hunk 3]
```diff
@@ -226,6 +226,19 @@ class QuickbooksEndpoint < EndpointBase::Sinatra::Base
     result code, summary
   end
 
+  post '/get_payments' do
+    qbo_payment = QBIntegration::Payment.new(@payload, @config)
+    summary, page, since, code = qbo_payment.get()
+
```
Comment (gorandev) [inline quickbooks_endpoint.rb]: Indentation woes here as well. Not a big problem, just makes confusing to check whether methods are correctly closed, might lead to bugs.

Comment (msalo3) [reply quickbooks_endpoint.rb]: @gorandev eagle eyes! I'm glad you're as sadistic as I am for this kind of thing. I can't believe I didn't notice until now

**Pull request merged**